src/sentry/static/sentry/app/views/organizationEventsV2/index.jsx

```diff
@@ -1,20 +1,49 @@
 import React from 'react';
 import DocumentTitle from 'react-document-title';
+import PropTypes from 'prop-types';
 
 import {t} from 'app/locale';
 import SentryTypes from 'app/sentryTypes';
 import GlobalSelectionHeader from 'app/components/organizations/globalSelectionHeader';
 import {PageContent, PageHeader} from 'app/styles/organization';
 import PageHeading from 'app/components/pageHeading';
 import BetaTag from 'app/components/betaTag';
+import NavTabs from 'app/components/navTabs';
+import ListLink from 'app/components/links/listLink';
+
+import Events from './events';
+import {ALL_VIEWS, getCurrentView} from './utils';
 
 export default class OrganizationEventsV2 extends React.Component {
   static propTypes = {
-    organization: SentryTypes.Organization,
+    organization: SentryTypes.Organization.isRequired,
+    location: PropTypes.object,
   };
 
-  render() {
+  renderTabs() {
     const {organization} = this.props;
+    const currentView = getCurrentView(this.props.location.query.view);
+
+    return (
+      <NavTabs underlined={true}>
+        {ALL_VIEWS.map(view => {
+          const query = view.id === 'all' ? '' : `?view=${view.id}`;
+          return (
+            <ListLink
+              key={view.id}
+              to={`/organizations/${organization.slug}/events/${query}`}
```

Comment:
Should we use a query object here instead of stringifying the url params ourselves

Reply:
This only really takes one of the 2 values - "errors" or "csp" so I'm not too worried about this

Reply:
What about global header?

Reply:
Ah, will update

Reply:
Updated in 6a55b08